Tests/FluentAssertions.Shared.Specs/DateTimeAssertionSpecs.cs

```diff
@@ -792,6 +792,47 @@ public void Should_fail_with_descriptive_message_when_asserting_datetime_has_sec
                 .WithMessage("Expected second 1 because we want to test the failure message, but found 0.");
         }
 
+        [TestMethod]
+        public void Should_succeed_when_asserting_datetime_is_on_same_date()
```

Comment:
Happily. Was not sure which convention to follow from `DateTimeAssertionSpecs` and CONTRIBUTING.md referenced a now-extinct file. I'll meet you later for a CONTRIBUTING.md PR.